torch/_dynamo/symbolic_convert.py

```diff
@@ -4655,6 +4670,7 @@ def __init__(self, *args: Any, **kwargs: Any) -> None:
         self.generated_items = []
         self.generator_exhausted = False
         self.is_generator_from_ctx_manager = False
+        self.latest_bytecode_queue = Queue
```

Comment:
Fix (also need to set `latest_bytecode_queue` of `InliningInstructionTranslator` as well?)